modules/tinymce/src/themes/silver/main/ts/ui/window/SilverDialogInstanceApi.ts

```diff
@@ -152,6 +156,7 @@ const getDialogApi = <T extends Dialog.DialogData>(
     setData,
     setEnabled,
     focus,
+    focusSelf,
```

Comment:
ooh, hang on. This is adding a public API, which has implications including documentation.

@spocke should we discuss this?